src/graphdb.js

```diff
@@ -193,34 +247,37 @@ function graphDBInferenceSchema (graphbSchema, addMutations) {
     // query
     r += `type Query {\n`;
     gdbs.nodeStructures.forEach(node => {
-        let nodeCase = toPascalCase(node.label); 
+        let nodeCase = cleanseLabel(node.label);
+        nodeCase = toPascalCase(nodeCase);
         r += `\tgetNode${nodeCase}(filter: ${nodeCase}Input): ${nodeCase}\n`;
         r += `\tgetNode${nodeCase}s(filter: ${nodeCase}Input, options: Options): [${nodeCase}]\n`;
     });
     r += '}\n\n';
 
     // mutation
-    if (addMutations) {        
+    if (addMutations) {
         r += `type Mutation {\n`;
         gdbs.nodeStructures.forEach(node => {
-            let nodeCase = toPascalCase(node.label);         
+            let nodeCase = cleanseLabel(node.label);
```

Comment:
Since you condensed other places to do `toPascalCase(cleanseLabel(X))` you can do that here as well.